static/scss/auth.scss

```diff
@@ -20,6 +20,58 @@
       font-size: 16px;
     }
   }
+
+  .confirm-sent-page {
+    font-size: 16px;
+    letter-spacing: 1.05px;
+    line-height: 20px;
+    padding: 49px 100px 8px;
+    position: relative;
+    box-shadow: inset 0 1px 3px 0 rgba(0,0,0,0.5), -4px 3px 17px -4px rgba(0,0,0,0.25);
+    background-color: $very-light-gray;
+
+    &:before {
+      position: absolute;
+      left: 0;
+      top: 0;
+      right: 0;
+      content: '';
+      height: 8px;
+      background: linear-gradient(to right, rgba(61,189,191,1) 0%,rgba(241,186,70,1) 100%);
+    }
+
+    a {
+      color: $blue;
```

Comment:
nit: $light-blue

Reply:
inspected from the design and come to know that actual color is `#126F9A` which we have as `$link-blue;`